### PR TITLE
UX Improvements - 280717

### PR DESCRIPTION
**Pull request by harpalsingh**
# Description

This update provides some new consistencies for the font and various components.

- Icon Font additions
- Carbon site font updates (show all)
- Heading / Dialog Improvements
- Button Toggle Improvements
- DatePicker Improvements

# TODO
- [x] Release notes

# Screenshots / Gifs

Visuals of the updated components

## DatePicker

As per original designs now. Larger buttons/text and cleaner visual style.

Now:
screen shot 2017-08-01 at 13 25 14

## Headings / Dialog

Font weight 900 applied to headings and correct spacing for subtitle and lines. Dialogs also get extra padding around the edge.

### Default
screen shot 2017-08-01 at 13 46 58

### With Subtitle
screen shot 2017-08-01 at 13 47 07

## Font

Visuals updated in Carbon site to show all icons. The visual below just shows 1 segment.

screen shot 2017-08-01 at 00 46 50

## Button Toggle

With weight 900 and darker text for selectable action. The hover color now matches default button hover color.

screen shot 2017-08-03 at 10 35 29

# Testing Instructions

### CODE REVIEW

**Comment by andrewjtait** [comment]
screen shot 2017-08-17 at 14 49 55

scrolling in dialog cuts off the bottom too early. Should the scrollbar also be on the edge of the dialog?

**Comment by lukeemmerson** [comment]
@harpalsingh Not sure if this bug is introduced with your padding update, but the sticky bottom bar on dialogs is too wide for the dialog it sits on top of. Also scrollbar issue Andrew mentioned. I wonder if we can sticky this to the right so that the padding doesn't affect it.

screen shot 2017-08-17 at 17 11 05

**Comment by lukeemmerson** [comment]
With the new toggle colours, we need to also consider icon colour. Because of the increase in contrast, i'd recommend turning the icon white on hover and active state. screen shot 2017-08-17 at 17 15 58

**Comment by lukeemmerson** [comment]
@harpalsingh Upon selecting the new carbon date picker in GAC it appears out of alignment when active.
screen shot 2017-08-17 at 17 19 27

**Comment by harpalsingh** [comment]
@lukeemmerson The datepicker alignment is a known issue and not impacted by the style change.

screen shot 2017-08-18 at 12 51 18

I would say that is an entirely different ticket to resolve this problem as the way the render is done, you can not know how far to the left it should appear.

**Comment by harpalsingh** [comment]
@lukeemmerson the icon issue in button toggle is not connected to this PR, that is stock work that has been overridden in GAC.

**Comment by harpalsingh** [comment]
@andrewjtait @lukeemmerson dialog issues addressed and updated into branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
+
+The following components have had styling updates:-
+
+* DatePicker
+* ButtonToggle
+* Heading / Dialog
+
+## Font Update
+
+The 300 weight (Thin) has been replaced by the 900 weight (Black) in Lato.
```
Comment (andrewjtait) [inline CHANGELOG.md]: this needs to go under 1.5.0

**Comment by andrewjtait** [comment]
Tested in  GAC and demo site in Chrome, Safari and Edge ✅

File: CHANGELOG.md (hunk 1 above)
Comment (harpalsingh) [inline CHANGELOG.md]: Done 👍

**Comment by lukeemmerson** [comment]
@harpalsingh Tested in GAC with Safari, Chrome and Firefox no visual bugs that i can see.

**Pull request merged**